DDG4/plugins/Geant4OpticalPhotonPhysics.cpp

```diff
@@ -81,11 +86,18 @@ namespace dd4hep {
         G4OpRayleigh*         fRayleighScatteringProcess = new G4OpRayleigh();
         G4OpMieHG*            fMieHGScatteringProcess    = new G4OpMieHG();
 
+#if G4VERSION_NUMBER >= 1070
+        G4OpticalParameters* params = G4OpticalParameters::Instance();
+        params->SetAbsorptionVerboseLevel(m_verbosity);
+        params->SetRayleighVerboseLevel(m_verbosity);
+        params->SetMieVerboseLevel(m_verbosity);
+        params->SetBoundaryVerboseLevel(m_verbosity);
```

Comment:
Could you add the boundaryInvokeSD parameter setting to this?

https://github.com/Geant4/geant4/blob/6399a014b611a4aa09ecba225418222dac261ca1/source/processes/electromagnetic/utils/src/G4OpticalParameters.cc#L129

Reply:
Implemented in 8e07cb7f9aa3b1c67aa607ab6bd6dbf6a0bcdd7a.